scripts/travis/build.sh

```diff
@@ -110,6 +110,10 @@ if [[ "${WITH_PYTHON}" == "true" ]]; then
     make pyarb -j4                                                           || error "building pyarb"
     progress "Python unit tests"
     python$PY $python_path/test/unit/runner.py -v2                           || error "running python unit tests (serial)"
+    python$PY $python_path/example/network_ring.py                           || error "running python network_ring example"
```

Comment:
maybe make use of https://github.com/arbor-sim/arbor/pull/1181 ?

Reply:
Is there an advantage to doing it that way?

Reply:
it would test that the examples can be used as "standalone" executables